src/components/scopedVariables/types.tsx

```diff
@@ -5,6 +5,16 @@ export enum ReadFileAs {
     ARRAY_BUFFER = 'arrayBuffer',
 }
 
+export enum FileView {
```

Comment:
Can optimize ConfigurationType from constant

Reply:
Actually this enum is supposed to restrict the type of state to two views only, so for better separation of concerns I haven't used this.